assets/src/design-system/components/contextMenu/menuItem.js

```diff
@@ -187,6 +201,7 @@ export const MenuItemProps = {
   onDismiss: PropTypes.func,
   onFocus: PropTypes.func,
   shortcut: PropTypes.string,
+  Icon: PropTypes.func,
```

Comment:
I think this should be `PropTypes.node` since it is renderable.
```suggestion
  Icon: PropTypes.node,
```

Reply:
actually, it's a function! i tried this first and got yelled at by the console. when you do `typeof Icon` it's a function.